static/styles/dark_theme.css

```diff
@@ -635,6 +635,11 @@ body.dark-theme {
 
     #recent_topics_table {
         border-color: hsla(0, 0%, 0%, 0.6);
+
+        .fa-envelope,
+        .fa-group {
+            color: hsl(221, 13%, 71%);
```

Comment:
This color does not appear anywhere else in the app; how did you pick it? Could the colors for these icons be represented as just an `opacity` on the default text color for the light/dark themes?

Reply:
@timabbott I have converted the color of both the icons to `opacity` in dark as well as light mode.